[Staging] Ingest Fails with S3 Error

```
17/06/15 14:26:03 ERROR Ingest$: geotrellis.spark.io.package$AttributeNotFoundError: Attribute ingestComplete not found for layer Layer(name = "30ab4c73-6b70-44f2-9f4c-84418ed1c194", zoom = 0)
	at geotrellis.spark.io.s3.S3AttributeStore.read(S3AttributeStore.scala:73)
	at com.azavea.rf.batch.util.Validation$.validateS3CatalogEntry(Validation.scala:22)
	at com.azavea.rf.batch.util.Validation$.validateCatalogEntry(Validation.scala:44)
	at com.azavea.rf.batch.util.Validation$.validateCatalogEntry(Validation.scala:39)
	at com.azavea.rf.batch.ingest.spark.Ingest$$anonfun$main$1.apply(Ingest.scala:323)
	at com.azavea.rf.batch.ingest.spark.Ingest$$anonfun$main$1.apply(Ingest.scala:323)
	at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:186)
	at com.azavea.rf.batch.ingest.spark.Ingest$.main(Ingest.scala:323)
	at com.azavea.rf.batch.ingest.spark.Ingest.main(Ingest.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.spark.deploy.yarn.ApplicationMaster$$anon$2.run(ApplicationMaster.scala:637)
Caused by: com.azavea.shaded.amazonaws.services.s3.model.AmazonS3Exception: The specified key does not exist. (Service: Amazon S3; Status Code: 404; Error Code: NoSuchKey; Request ID: 17D3B6CDC601512A), S3 Extended Request ID: VrzqmsS9ka5+zbbeFCEMK+e2AWc6NekpHNyGKJFyiiuUq9TlVkKhrfQsMZWBkRPK2/K1Hx3nwvY=
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1579)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1249)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1030)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:742)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:716)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:699)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:667)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:649)
	at com.azavea.shaded.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:513)
	at com.azavea.shaded.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4194)
	at com.azavea.shaded.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4141)
	at com.azavea.shaded.amazonaws.services.s3.AmazonS3Client.getObject(AmazonS3Client.java:1384)
	at geotrellis.spark.io.s3.AmazonS3Client.getObject(AmazonS3Client.scala:61)
	at geotrellis.spark.io.s3.S3Client$class.getObject(S3Client.scala:70)
	at geotrellis.spark.io.s3.AmazonS3Client.getObject(AmazonS3Client.scala:43)
	at geotrellis.spark.io.s3.S3AttributeStore.geotrellis$spark$io$s3$S3AttributeStore$$readKey(S3AttributeStore.scala:57)
	at geotrellis.spark.io.s3.S3AttributeStore.read(S3AttributeStore.scala:70)
	... 14 more
```

--- a/app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala
+++ b/app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala
@@ -320,15 +320,13 @@ object Ingest extends SparkJob with LazyLogging with Config {
     implicit def asS3Payload(status: IngestStatus): String = S3IngestStatus(sceneId, status).asJson.noSpaces
 
     try {
-      if (params.testRun) { ingestDefinition.layers.foreach(Validation.validateCatalogEntry) }
-      else {
-        ingestDefinition.layers.foreach(ingestLayer(params))
-        putObject(
-          params.statusBucket,
-          ingestDefinition.id.toString,
-          IngestStatus.Ingested
-        )
-      }
+      ingestDefinition.layers.foreach(ingestLayer(params))
+      if (params.testRun) ingestDefinition.layers.foreach(Validation.validateCatalogEntry)
+      putObject(
+        params.statusBucket,
+        ingestDefinition.id.toString,
+        IngestStatus.Ingested
+      )
     } catch {
       case t: Throwable =>
         logger.error(t.stackTraceString)

--- a/app-tasks/dags/ingest_project_scenes.py
+++ b/app-tasks/dags/ingest_project_scenes.py
@@ -43,7 +43,7 @@ dag = DAG(
 batch_job_definition = os.getenv('BATCH_INGEST_JOB_NAME')
 batch_job_queue = os.getenv('BATCH_INGEST_JOB_QUEUE')
 hosted_zone_id = os.getenv('HOSTED_ZONE_ID')
-jar_path = os.getenv('BATCH_JAR_PATH', 'rf-batch-g874b19.jar')
+jar_path = os.getenv('BATCH_JAR_PATH', 'rf-batch-94265f7.jar')
 
 
 ################################

